src/app/lesson/page.tsx

```
@@ -1,23 +1,41 @@
-
+import ReactMarkdown, {Components} from "react-markdown";
+import Image from "next/image";
 import "./lesson.css";
 import Footer from "@/app/components/Footer";
 import { useSearchParams } from "next/navigation";
 import axios from "axios";
 import { API_URL } from "../../../lib/config";
 import refreshSession from "../../../lib/refreshSession";
-
+import rehypeRaw from "rehype-raw";
+import { Warning, Note, Error, Props } from "../components/MarkdownCards";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)